src/App.css

```
@@ -1,38 +1,68 @@
-.App {
-  text-align: center;
+body{
+    height: 100vh;
+    margin: 0;
+    background-color: #eeeeee;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+
+
+#root, .App{
+    height: 100vh;
+    background-color: #eeeeee;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+
+.top-bar{
+    height: 7vh;
+    box-sizing: border-box;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+.top-bar h1{
+    padding-top: 2px;
+    padding-bottom: 2px;
+    padding-left: 10px;
```

```suggestion
    padding: 0.8rem
```
Try this and let me know your thoughts